universe/common/api/middleware/verify_user.go

```diff
@@ -7,27 +7,24 @@ import (
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
 
-	"github.com/momentum-xyz/ubercontroller/types/entry"
-	"github.com/momentum-xyz/ubercontroller/universe"
 	"github.com/momentum-xyz/ubercontroller/universe/common/api"
-	"github.com/momentum-xyz/ubercontroller/utils"
 )
 
 func VerifyUser(log *zap.SugaredLogger) gin.HandlerFunc {
-	return func(c *gin.Context) {
-		// get jwt secret
-		jwtSecret, ok := universe.GetNode().GetNodeAttributeValue(
-			entry.NewAttributeID(universe.GetSystemPluginID(), universe.Attributes.Node.JWTKey.Name),
-		)
-		if !ok || jwtSecret == nil {
-			err := errors.New("Middleware: VerifyUser: failed to get jwt_key attribute")
-			api.AbortRequest(c, http.StatusInternalServerError, "failed_to_get_jwt_key", err, log)
-			return
+	var secret []byte = nil
+	if secret == nil {
+		jwtSecret, err := api.GetJWTSecret()
+		if err != nil {
+			err = errors.WithMessage(err, "failed to fetch jwt secret")
```

Comment:
please add "Middleware: VerifyUser: " as error prefix to clearly understand which particular part of application failed